Crash of the vppagent-icmp-responder

## Current Behavior

The vppagent-icmp-responder has crashed on startup

## Context

nsm commit id used to build the vppagent-icmp-responder image: 
4288a5288c935e8a48bd5aadb9f395567677cc89

## Failure Logs

2019-07-01 14:23:33,155 DEBG 'application' stdout output:
time="2019-07-01T14:23:33Z" level=info msg="NSE registered: network_service:<name:\"icmp-responder\" payload:\"IP\" > network_service_manager:<name:\"kube-master\" url:\"10.32.0.6:5001\" last_seen:<seconds:1561385273 > state:\"RUNNING\" > networkservice_endpoint:<network_service_name:\"icmp-responder\" payload:\"IP\" network_service_manager_name:\"kube-master\" endpoint_name:\"icmp-responderkmmp5\" labels:<key:\"app\" value:\"icmp-responder\" > labels:<key:\"networkservicename\" value:\"icmp-responder\" > state:\"RUNNING\" > "
time="2019-07-01T14:23:33Z" level=info msg="NSE: channel has been successfully advertised, waiting for connection from NSM..."

2019-07-01 14:23:38,160 DEBG 'application' stdout output:
time="2019-07-01T14:23:38Z" level=info msg="Request for Network Service received connection:<id:\"21\" network_service:\"icmp-responder\" context:<src_ip_required:true dst_ip_required:true excluded_prefixes:\"10.32.0.0/12\" excluded_prefixes:\"10.96.0.0/12\" > labels:<key:\"app\" value:\"icmp\" > > mechanism_preferences:<type:MEM_INTERFACE > mechanism_preferences:<type:KERNEL_INTERFACE > "

2019-07-01 14:23:38,161 DEBG 'application' stdout output:
time="2019-07-01T14:23:38Z" level=info msg="New connection created: id:\"21\" network_service:\"icmp-responder\" mechanism:<type:MEM_INTERFACE parameters:<key:\"description\" value:\"NSM Endpoint\" > parameters:<key:\"name\" value:\"nsmjehI0hp5\" > parameters:<key:\"netnsInode\" value:\"4026532876\" > parameters:<key:\"socketfile\" value:\"nsmjehI0hp5/memif.sock\" > > context:<src_ip_required:true dst_ip_required:true excluded_prefixes:\"10.32.0.0/12\" excluded_prefixes:\"10.96.0.0/12\" > "
2019/07/01 14:23:38 Reporting span 2430dfc222b881e0:5c31a66d6d3eecc0:462df5b23bbd8fe0:1

2019-07-01 14:23:38,164 DEBG 'application' stdout output:
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x8bd304]

goroutine 55 [running]:
github.com/networkservicemesh/networkservicemesh/controlplane/pkg/prefix_pool.intersect(0xc000247740, 0x0, 0x0)
	/root/networkservicemesh/controlplane/pkg/prefix_pool/prefixpool.go:277 +0x54
github.com/networkservicemesh/networkservicemesh/controlplane/pkg/prefix_pool.(*prefixPool).ExcludePrefixes(0xc00019d180, 0xc000099be0, 0x2, 0x2, 0x0, 0x0, 0x0
2019-07-01 14:23:38,165 DEBG 'application' stdout output:
, 0x0, 0x0)
	/root/networkservicemesh/controlplane/pkg/prefix_pool/prefixpool.go:90 +0x6e7
github.com/networkservicemesh/networkservicemesh/sdk/endpoint.(*IpamEndpoint).Request(0xc00000b440, 0xba0f80, 0xc000247440, 0xc0000b3700, 0xc00008e520, 0xc000188c80, 0x30)
	/root/networkservicemesh/sdk/endpoint/ipam.go:54 +0x190
main.(*vppagentComposite).Request(0xc0001bc930, 0xba0f80, 0xc000247440, 0xc0000b3700, 0xb12053, 0x27, 0xc0002077e0)
	/root/networkservicemesh/test/applications/cmd/vppagent-icmp-responder-nse/composite.go:49 +0x79
github.com/networkservicemesh/networkservicemesh/sdk/endpoint.(*MonitorEndpoint).Request(0xc00000a6c0, 0xba0f80, 0xc000247440, 0xc0000b3700, 0xc0002077e0, 0x1, 0x1)
	/root/networkservicemesh/sdk/endpoint/monitor.go:47 +0x78
github.com/networkservicemesh/networkservicemesh/sdk/endpoint.(*CompositeEndpoint).Request(0xc00000b560, 0xba0f80, 0xc000247440, 0xc0000b3700, 0x1, 0xa41ba0, 0xab33c0)
	/root/networkservicemesh/sdk/endpoint/composite.go:64 +0x7e
github.com/networkservicemesh/networkservicemesh/sdk/endpoint.(*nsmEndpoint).Request(0xc0002525a0, 0xba0f80, 0xc000247440, 0xc0000b3700, 0xc0002525a0, 0x0, 0x0)
	/root/networkservicemesh/sdk/endpoint/endpoint.go:153 +0xc1
github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/local/networkservice._NetworkService_Request_Handler.func1(0xba0f80, 0xc000247440, 0xacd820, 0xc0000b3700, 0xba0f80, 0xc000247440, 0xba5000, 0xc0001541e0)
	/root/networkservicemesh/controlplane/pkg/apis/local/networkservice/networkservice.pb.go:168 +0x86
github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc.OpenTracingServerInterceptor.func1(0xba0f80, 0xc000247440, 0xacd820, 0xc0000b3700, 0xc000099c20, 0xc000099c40, 0x0, 0x0, 0x0, 0x0)
	/go/pkg/mod/github.com/grpc-ecosystem/grpc-opentracing@v0.0.0-20180507213350-8e809c8a8645/go/otgrpc/server.go:57 +0x2b0
github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/local/networkservice._NetworkService_Request_Handler(0xab33c0, 0xc0002525a0, 0xba0f80, 0xc000247290, 0xc0002f1180, 0xc0002ac260, 0x0, 0x0, 0xc0000e6410, 0x4d)
	/root/networkservicemesh/controlplane/pkg/apis/local/networkservice/networkservice.pb.go:170 +0x158
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001a9c80, 0xba4620, 0xc0002e2600, 0xc00011a600, 0xc000270ff0, 0x1055e00, 0x0, 0x0, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.21.1/server.go:998 +0x4a2
google.golang.org/grpc.(*Server).handleStream(0xc0001a9c80, 0xba4620, 0xc0002e2600, 0xc00011a600, 0x0)
	/go/pkg/mod/google.golang.org/grpc@v1.21.1/server.go:1278 +0xe02
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc000039190, 0xc0001a9c80, 0xba4620, 0xc0002e2600, 0xc00011a600)
	/go/pkg/mod/google.golang.org/grpc@v1.21.1/server.go:717 +0x9f
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/go/pkg/mod/google.golang.org/grpc@v1.21.1/server.go:715 +0xa1

--- a/k8s/conf/vppagent-icmp-responder-nse.yaml
+++ b/k8s/conf/vppagent-icmp-responder-nse.yaml
@@ -41,7 +41,7 @@ spec:
             - name: TRACER_ENABLED
               value: "true"
             - name: IP_ADDRESS
-              value: "10.30.1.1"
+              value: "10.30.1.0/24"
           resources:
             limits:
               networkservicemesh.io/socket: 1

